packages/frontend/src/main/pages/stream/CommitObjectViewer.vue

```diff
@@ -546,6 +572,9 @@ export default defineComponent({
     }, 300)
   },
   methods: {
+    disableScroll() {
+      // TODO
```

Comment:
There are some unfinished methods and commented out templates in this PR, should they be there?